### ISSUE TITLE
User Manager:: Inconsistency in 'Success message', normally in joomla we use '1' instead of 'One' for success message

### ISSUE THREAD

**Issue by aasimali** [joomla/joomla-cms#5636]
-Go to 'User Manager>User'
-Unblock or block  a user
-Check the message.
-Message should be:- **1 user unblocked/blocked**
![screen shot 2015-01-07 at 05 14 49](http://issues.joomla.org/uploads/1/72cb0044f03b9e5b9ca43746ba022e14.png)
-------------------------

-Go to 'User Manager>User Group'
-Delete a user group
-Check the message.
-Message should be:- **1 User Group successfully deleted**
![screen shot 2015-01-07 at 05 16 46](http://issues.joomla.org/uploads/1/e45952c7c2469908cda74f6f918df1c9.png)

**Comment by brianteeman** [comment]
This is being looked at as part of the en-GB test string standardisation
issue - thanks

On 7 January 2015 at 11:21, Aasim Ali <notifications@github.com> wrote:

> -Go to 'User Manager>User'
> -Unblock or block a user
> -Check the message.
> -Message should be:- *1 user unblocked/blocked*
> [image: screen shot 2015-01-07 at 05 14 49]
> <https://camo.githubusercontent.com/ff3276001c03975c4f3bc18caf5696efb8b1688b/687474703a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f37326362303034346630336239653562396361343337343662613032326531342e706e67>
>
> -Go to 'User Manager>User Group'
> -Delete a user group
> -Check the message.
> -Message should be:- *1 User Group successfully deleted*
> [image: screen shot 2015-01-07 at 05 16 46]
> <https://camo.githubusercontent.com/d491479f96f043e975fe224a03f5886b613a0f46/687474703a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f65343539353263376332343639393038636461373466366639313864663163392e706e67>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/issues/5636>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by aasimali** [comment]
@brianteeman  You're welcome :)

**Comment by brianteeman** [comment]
For reference we are adopting the findings from here
http://www.nngroup.com/articles/web-writing-show-numbers-as-numerals/

The problem is that the way the messages are generated when there are more than one item then it is always a number and not a word.

So that means currently you would have an inconsistency where it would be
One item unpublished
22 items unpublished

So we are adopting the NNG findings and standardising on using numerals as documented here https://github.com/joomla/user-interface-text/blob/master/A-Z.md#numbers<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5636">issues.joomla.org/joomla-cms/5636</a>.</sub>

**Comment by aasimali** [comment]
@brianteeman 
I have reported 2 issue at a same time. Hope the second one will also be solved.(1st screenshot)
 **1/one user unblocked** should not be **1/one user enabled**

**Comment by brianteeman** [comment]
PLEASE report one issue only - it is impossible to manage if you report multiple unrelated issues at the same time. 

please create a new issue for the 1st screenshot.

**Comment by aasimali** [comment]
@brianteeman 
Okay! I am sorry. I will report one issue at a time.
-Reporting issue for 1st screenshot.
